pkg/proc/threads.go

```diff
@@ -399,6 +398,11 @@ func setStepIntoBreakpoint(dbp Process, text []AsmInstruction, cond ast.Expr) er
 
 	pc := instr.DestLoc.PC
 
+	// Skip InhibitStepInto functions for different arch.
+	if instr.IsCall() && dbp.BinInfo().Arch.InhibitStepInto(dbp.BinInfo(), pc) {
```

Comment:
No need to have `instr.IsCall()` here, it's always a call instruction.

Reply:
Ok, done.